src/Scriban/Parsing/Parser.Expressions.cs

```diff
@@ -660,6 +660,18 @@ private ScriptExpression ParseExpression(ScriptNode parentNode, ScriptExpression
                         break;
                     }
 
+                    if ((!_isScientific) && (parentNode is ScriptPipeCall) // after a pipe call we expect to see a function call
+                                         && (functionCall == null)         // but when function is not followed by any parameter e.g. '1 | math.abs', above code does not create function call,
+                                                                           // here we fix that by creating function call, but only when leftOperand is e.g. '1 | abs' or '1 | math.abs' 
+                                         && (leftOperand is ScriptVariableGlobal || leftOperand is ScriptMemberExpression) // we need that restriction since leftOperand can be of other type e.g. binary expression '"123" | string.to_int + 1'
```

Comment:
```suggestion
                                         && (leftOperand is IScriptVariablePath) // we need that restriction since leftOperand can be of other type e.g. binary expression '"123" | string.to_int + 1'
```